tutor/templates/build/openedx/Dockerfile

```diff
@@ -71,32 +112,126 @@ RUN cd /tmp \
+
+# Install nodejs requirements
+ARG NPM_REGISTRY={{ NPM_REGISTRY }}
+COPY --link --from=edx-platform /package.json package.json
+COPY --link --from=edx-platform /package-lock.json package-lock.json
+COPY --link --from=edx-platform /scripts/copy-node-modules.sh scripts/copy-node-modules.sh
+RUN --mount=type=cache,target=/root/.npm,sharing=shared \
+    npm clean-install --no-audit --registry=$NPM_REGISTRY
+
+# Set up static root.
+# By default, CMS static root is assumed to be $STUDIO_ROOT_LMS/studio.
+ENV STATIC_ROOT_LMS=/openedx/staticfiles
+RUN mkdir -p "$STATIC_ROOT_LMS"
+
+
+###########################################################################################################################
+############## BUNDLES
+###########################################################################################################################
+
+FROM frontend-requirements as bundles
+
+COPY --link --from=edx-platform /.babelrc .babelrc
+COPY --link --from=edx-platform /webpack-config/file-lists.js webpack-config/file-lists.js
+COPY --link --from=edx-platform /webpack.builtinblocks.config.js webpack.builtinblocks.config.js
+COPY --link --from=edx-platform /webpack.common.config.js webpack.common.config.js
+COPY --link --from=edx-platform /webpack.dev.config.js webpack.dev.config.js
+COPY --link --from=edx-platform /webpack.prod.config.js webpack.prod.config.js
+COPY --link --from=edx-platform /cms/djangoapps/pipeline_js/js cms/djangoapps/pipeline_js/js
+COPY --link --from=edx-platform /cms/static cms/static
+COPY --link --from=edx-platform /cms/templates/ cms/templates/
+COPY --link --from=edx-platform /common/static/common common/static/common
+COPY --link --from=edx-platform /common/static/js/ common/static/js
+COPY --link --from=edx-platform /lms/djangoapps/discussion/static lms/djangoapps/discussion/static
+COPY --link --from=edx-platform /lms/djangoapps/instructor/static lms/djangoapps/instructor/static
+COPY --link --from=edx-platform /lms/djangoapps/support/static/support lms/djangoapps/support/static/support
+COPY --link --from=edx-platform /lms/djangoapps/teams/static lms/djangoapps/teams/static
+COPY --link --from=edx-platform /lms/static/ lms/static/
+COPY --link --from=edx-platform /lms/templates/ lms/templates/
+COPY --link --from=edx-platform /openedx/features/announcements/static openedx/features/announcements/static
+COPY --link --from=edx-platform /openedx/features/course_bookmarks/static openedx/features/course_bookmarks/static
+COPY --link --from=edx-platform /openedx/features/course_experience/static openedx/features/course_experience/static
+COPY --link --from=edx-platform /openedx/features/course_search/static openedx/features/course_search/static
+COPY --link --from=edx-platform /openedx/features/learner_profile/static openedx/features/learner_profile/static
+COPY --link --from=edx-platform /xmodule/assets xmodule/assets
+COPY --link --from=edx-platform /xmodule/js xmodule/js
```

Comment:
Wooooooow... Are all "static" directories copied? Does it mean that every new edx-platform app will require a change in Tutor? If yes, that's an issue.

Same comment below for sass directories.

Reply:
@regisb

> Wooooooow... Are all "static" directories copied?

...yes 😬 

> Does it mean that every new edx-platform app will require a change in Tutor?

In theory no, since apps should not be adding static assets to edx-platform; all new frontends should be MFEs.

That said,  upstream refactorings or deprecations could definitely break this build stage. I agree it has far too many COPY statements.

I would still very much like to keep it so that they static asset build isn't re-run every time backend edx-platform code is changed. We need a way to get all of our frontend sources into a build stage without also copying backend assets, but we want to avoid hard-coding a big list of static directories.

I did some research, and here's what I came up with: https://github.com/kdmccormick/tutor/pull/35. Let me know what you think.

Reply:
Yes, #35 looks much more tractable.